api/src/services/fields.ts

```diff
@@ -405,6 +405,16 @@ export class FieldsService {
 							this.addColumnToTable(table, collection, hookAdjustedField as Field);
 						});
 					}
+
+					if (hookAdjustedField.schema) {
```

Comment:
This will always throw an error, since the fields record doesn't exist until L429 (when applicable).